Sources/JXKit/JXContext.swift

```diff
@@ -510,11 +513,16 @@ extension JXContext {
 
 /// Optional service provider integration points.
 public protocol JXContextSPI {
+    func eval(_ script: String, this: JXValue?, in: JXContext) throws -> JXValue?
```

Comment:
JXBridge doesn't do its own eval. But it does want a pre-eval hook, and this serves that purpose too.